web-extensions/chrome/utils.js

```diff
@@ -2,3 +2,10 @@
 function getBrowser() {
     return chrome || window.chrome || browser || window.browser;
 }
+
+/* eslint-disable-next-line no-unused-vars */
+function cloneArray(arr) {
+    // The "Array.prototype.slice.call" call is to prevent chrome very bad handling of DOM
```

Comment:
ok, this comment explains the why.
A bit sad that you have to add a custom linter because of this though. Could you not find a "kind-of standard" tool (on npm for instance) to check for "forbidden functions"?

Reply:
I intend to but I don't want to do it in the same PR.

Reply:
might be smaller/simpler than adding a linter for this, though :D